.changeset/rare-cooks-peel.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-user-settings': patch
+---
+
+Use feature flag description in user settings if available
```

Comment:
Hmm a bit unclear what this actually means for the user. Maybe something around: `Feature flag objects now accept a description prop`. Feel free to play around with it.